css/src/core/animations.scss

```diff
@@ -10,3 +10,23 @@
 		transform: rotate(359deg);
 	}
 }
+
+@keyframes boop {
+	0% {
+		transform: scale(1);
+	}
+
+	50% {
+		transform: scale(1.1);
+	}
+
+	100% {
+		transform: scale(1);
+	}
+}
+
+@mixin reduceMotion {
```

Comment:
Should be in mixins, if included at this point at all

Reply:
reverted